plugins/scala_test/tests/test_update_imports.py

```diff
@@ -0,0 +1,40 @@
+from logging import debug, error
+from pathlib import Path
+
+from os.path import join, basename
+from os import listdir
+
+from update_imports import update_imports
+# from update_imports import update_imports
+
+def test_update_imports():
+    summary = update_imports("plugins/scala_test/tests/resources/input/", dry_run=True)
+    assert is_as_expected("plugins/scala_test/tests/resources/", summary)
+
+def is_as_expected(path_to_scenario, output_summary):
```

Comment:
Wonder if there is a clean way to avoid the duplication between this code and the [top level test harness logic](https://github.com/uber/piranha/blob/b16fa19a340cba45a368adda64c90bf3468e2293/tests/tests.py#L198). Maybe a shared test utilities library? Not a big deal, but if every plugin will have it's own copy of this code that might be a pain when you need to update something.

Reply:
I agree. I wanted to that . I will eventually extract out a `commons` . 
This could moved to `commons/test_utilities`
I believe that `replace_imports` could also be a part of `commons/scala`